examples/docker-compose.yml

```diff
@@ -0,0 +1,332 @@
+# Note: If using ab-cli installer this file will get overwritten
+version: "3.9"
+
+volumes:
+  files:
+  mysql_data:
+  redis_data:
+  # clamav:
+
+networks:
+  default:
+    attachable: true
+
+services:
+
+  #nginx setup
+  web:
+    image: docker.io/digiserve/ab-web:$AB_WEB_VERSION
+    hostname: web
+    volumes:
+      - files:/data
+    depends_on:
+      - api_sails
+  #/nginx
+
+
+  #db: use Maria DB as our backend DB
+  db:
+    image: docker.io/digiserve/ab-db:$AB_DB_VERSION
+    hostname: db
+    environment:
+      MYSQL_ROOT_PASSWORD: $MYSQL_PASSWORD
+    volumes:
+      # - mysql_config:/etc/mysql/conf.d
+      - mysql_data:/var/lib/mysql
+      # - mysql_key:/key
+    # on windows: use this command. (be sure to clear out mysql/data folder)
+    # command: mysqld --innodb-flush-method=littlesync --innodb-use-native-aio=OFF --log_bin=ON
+    ######
+    # when there is a problem with : Error: ER_CRASHED_ON_USAGE: Table 'AAAAAA' is marked as crashed and should be repaired
+    # this can happen with the alter table algorithm: try the safest(and slowest) COPY
+    # command: ["mysqld", "--alter-algorithm=copy"]
+  #/db
+
+
+  #redis: use redis to allow cote services to find each other across a swarm
+  redis:
+    image: docker.io/redis
+    ## NOTE: don't expose the service to the outside for security:
+    ## if you do expose it, then set a password.
+    hostname: redis
+    volumes:
+      - redis_data:/data
+    command: redis-server --appendonly yes
+  #/redis
+
+
+  #api_sails: our API end point
+  api_sails:
+    image: docker.io/digiserve/ab-api-sails:$AB_API_SAILS_VERSION
+    # workaround for SELinux bind mount issue (for podman only)
+    # security_opt:
+    #   label: disable
+    environment:
+      - COTE_DISCOVERY_REDIS_HOST=redis
+      - NODE_ENV
+      - MYSQL_PASSWORD
+      # - MYSQL_HOST
+      # - MYSQL_PORT
+      # - MYSQL_USER
+      # - MYSQL_DBPREFIX
+      # - MYSQL_DBADMIN
+      # - MYSQL_POOL_MAX
+      # - MYSQL_POOL_ACQUIRE_TIMEOUT
+      - CAS_ENABLED
+      - CAS_BASE_URL
+      - CAS_UUID_KEY
+      - SITE_URL
+      - OKTA_ENABLED
+      - OKTA_DOMAIN
+      - OKTA_CLIENT_ID
+      - OKTA_CLIENT_SECRET
+      # - FILE_PROCESSOR_PATH
+      # - FILE_PROCESSOR_UPLOAD_DIR
+      # - FILE_PROCESSOR_MAXBYTES
+      - RELAY_ENABLED
+      - RELAY_ENABLE
```

Comment:
Do we actually need both of these `RELAY_*` entries?  Seems like we could simplify this to one of the other.  Or if they represent different settings ... we should rename these to reflect the purpose better.